### ISSUE TITLE
Whitespace issues

### ISSUE THREAD

**Issue by FloricSpacer** [FloricSpacer/AbyssDiver#10]
So, it seems like the conversion to Tweego has altered the intentional whitespace I placed at the beginning and end of each passage. I think it looks aesthetically better to have an extra line break at the top and bottom of each passage, so the words don't conflict with the background. 

Unfortunately, it seems that Tweego ignores additional line breaks at the beginning and end of each passage, so I may have to go through and add <br> line breaks manually to the beginning and end of each passage. @emmeken Is there a setting to flip on Tweego to make it not ignore additional line breaks? If not, I'll go through it manually to fix each passage. 

I'd also encourage all new passages (that players will view, so not time passages or similar) to have <br> at the beginning and end so that the spacing is consistent. Unless you have some variable you set, which can also serve as the proper line break.

**Comment by emmeken** [comment]
I think we should fix the layout by setting margins in CSS instead of adding blank lines to the HTML.

**Comment by emmeken** [comment]
I committed a CSS change that adds padding (not margins) on the `#passages` element. Feel free to tweak the actual amount of whitespace if you'd prefer a bit more or less.

The passage text should not have empty lines at the start or end in this approach. I modified the layer 1 hub, but there will likely be many more with empty lines at the start that can be removed. We can address those over time, I think.
